performance/multi-node/rdm_mbw_mr.c

```diff
@@ -536,18 +647,34 @@ int main(int argc, char *argv[])
 	if (!hints)
 		return -1;
 
-	while ((op = getopt(argc, argv, "hp:w:vr:" CT_STD_OPTS)) != -1) {
+	while ((op = getopt(argc, argv, "hx:p:w:vr:m:c:" CT_STD_OPTS)) != -1) {
 		switch (op) {
 		default:
 			ct_parse_std_opts(op, optarg, hints);
 			break;
-		case 'p':
+		case 'x':
```

Comment:
p is already the standard for change the provider type, and when you tried to use -p sockets the test would use sockets but then set pairs to something undefined. Switching to x allows the ct_parse_std_opts to work and the pair argument to work.